middleware/1.auth.ts

```diff
@@ -0,0 +1,34 @@
+import { checkPrivy } from '../utils/checkPrivy'
+
+export default eventHandler(async (event) => {
```

Comment:
How come this reads `eventHandler` vs. `defineEventHandler`? I think we can change it to match the docs and suggested available auto imports.